app/helpers/calendar_helper.rb

```diff
@@ -0,0 +1,49 @@
+module CalendarHelper
+  def calendar(date = Date.today, &block)
+    Calendar.new(self, date, block).table
+  end
+
+  class Calendar < Struct.new(:view, :date, :callback)
+    HEADER = %w(Sunday Monday Tuesday Wednesday Thursday Friday Saturday)
```

Comment:
Hound is acting weird here, there are no mutable objects here.
`%w(Sunday Monday ..)` returns `["Sunday", "Monday", ...]`
Hound Fucked up!